lib/commands/connect/regions.js

```diff
@@ -0,0 +1,50 @@
+'use strict';
+const cli = require('heroku-cli-util');
+
+const prodRegions = ['us', 'eu', 'tokyo', 'virginia', 'oregon', 'frankfurt', 'sydney'];
```

Comment:
So, we could also just *not* validate what the user provides us in `--region` and then try to error gracefully after that.

We could also cache the region listing, potentially, from the API on disk and refresh it when the cache-control headers from Heroku have expired. That would likely necessitate a new library dependency which the CLI team strongly advises against.